.github/workflows/dev_on_pull_request_run_e2e_web.yml

```
@@ -0,0 +1,112 @@
+
+name: '[on_pr] Run E2E smoke tests on Web'
+
+on:
+  push:
+    branches:
+        - PC-35750
+
+permissions:
+  contents: read
+  id-token: write
+  pull-requests: write
+
+jobs:
+  run_e2e_web:
+    name: 'Smoke test Staging Web'
+    runs-on: macos-15
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+      - name: Enable Corepack
+        run: corepack enable
+      - name: Setup java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - uses: pass-culture-github-actions/cache@v1.0.0
+        id: yarn-modules-cache
+        with:
+          path: |
+            node_modules
+            ~/.cache/yarn
+          key: v1-yarn-dependency-cache-${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            v1-yarn-dependency-cache-${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+          bucket: ${{ secrets.CACHE_BUCKET_NAME }}
+          workload-identity-provider: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
+          service-account: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
+      - name: Yarn install
+        run: yarn install
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Set Maestro Version
+        run: echo "MAESTRO_VERSION=1.40.0" >> $GITHUB_ENV
+
+      - name: Cache Maestro CLI
+        id: maestro-cache
+        uses: actions/cache@v4
+        with:
+          path: ~/.maestro
+          key: ${{ runner.os }}-maestro-${{ env.MAESTRO_VERSION }}
+          restore-keys: |
+            ${{ runner.os }}-maestro-
+
+      - name: Install Maestro CLI
+        if: steps.maestro-cache.outputs.cache-hit!= 'true'
+        run: |
+          echo "Installing Maestro CLI version ${MAESTRO_VERSION}..."
+          curl -Ls "https://get.maestro.mobile.dev" | bash
+          echo "$HOME/.maestro/bin" >> $GITHUB_PATH
+        env:
+          MAESTRO_VERSION: ${{ env.MAESTRO_VERSION }}
+
+      - name: Add Maestro to PATH (from cache)
+        if: steps.maestro-cache.outputs.cache-hit == 'true'
+        run: |
+          echo "Using cached Maestro CLI."
+          echo "$HOME/.maestro/bin" >> $GITHUB_PATH
+          maestro -v # Verify version
+
+      - name: Start Vite server
+        run: nohup yarn start:web:staging --host 0.0.0.0 > vite.log 2>&1 &
```

ça sert à quoi de mettre les logs dans un fichier alors qu'on n'en fait rien ?

si on veut juste cacher les logs, on peut faire ça

```suggestion
        run: nohup yarn start:web:staging --host 0.0.0.0 >/dev/null 2>&1 &
```